2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -589,4 +589,63 @@ Nell'attività di controllo si intraprendono azioni correttive nei confronti dei
 È necessario che alla chiusura dello sprint le attività ancora in fase di svolgimento vengano riportate nello sprint successivo, insieme a tutte quelle attività pianificate ma non ancora iniziate che sono considerate importanti. Tutte le attività che non vengono considerate importanti, ad esempio attività di cui si è rivalutato il grado di priorità, vengono riportate nel backlog. Una situazione di mancato raggiungimento degli obiettivi può essere sinonimo anche da una pianificazione errata e troppo ottimista, ed è quindi necessario che essa sia rivista e migliorata.
 
 ===== Miglioramenti e accortezze da adottare
-In questo caso le soluzioni correttive vengono decise dal gruppo tramite la visualizzazione e l'analisi della board Miro durante la retrospettiva. Nella board infatti, come esposto nella @valutazione, alcune delle task raccolte rispondono direttamente ai problemi individuati nella parte di Things to change.
\ No newline at end of file
+In questo caso le soluzioni correttive vengono decise dal gruppo tramite la visualizzazione e l'analisi della board Miro durante la retrospettiva. Nella board infatti, come esposto nella @valutazione, alcune delle task raccolte rispondono direttamente ai problemi individuati nella parte di Things to change.
+
+
+== Processo di gestione delle Decisioni 
+_Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.3.3_
+=== Scopo
+Lo scopo del processo di gestione delle decisioni è fornire un quadro strutturato per identificare, categorizzare e valutare le decisioni che si intendono intraprendere, selezionandone la migliore in termini di benefici attesi e costi e rischi associati. \
+
+==== Presentazione delle decisioni
+Ogni decisione viene presentata identificandone:
+- tipologia;
+- obiettivo;
+- soluzione proposta;
+- vantaggi;
+- svantaggi;
+- impatto in termini di tempi e costi;
+
+Il processo di decision making viene prevalentemente svolto nel meeting di retrospettiva, in modo da non contrastare la pianificazione dello sprint in corso ed evitando un eccessivo numero di meeting interni che potrebbero comportare difficoltà organizzative e un rallentamnto dell'avanzamento. 
+Solo in caso di decisioni critiche che riguardo cambiamenti sostanziali o nuove direzioni di lavoro possono far scaturire meeting interni mirati.
```

Comment:
Riformulerei in "Solo decisioni critiche riguardo cambiamenti sostanziali o nuove direzioni di lavoro possono far scaturire meeting interni mirati"